### PR TITLE
OP-707_OP-721 Introduce OPD ward

### PR DESCRIPTION
**Pull request by mwithi**
See OP-707 and OP-721.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -259,15 +261,17 @@ The OPD Extended module affords the following additional features:
...
 * Once the patient is selected and a previous OPD visit is found, special fields are populated with the most recent visit of the patient (4), his/her personal data (5), and the Re-Attendance checkbox will be selected automatically upon selection of the main (first) disease (6);
-* It is now possible to set the next appointment date (7). This automatically schedules a visit visible in *_Therapy Management_*.
-* if a new attendance for this patient is required set the "New Admittance" check box and proceed normally.
+* It is now possible to set the next appointment date (7) and in what ward (OPD or IPD). This automatically schedules a visit visible in <<therapy-management,Therapy Management>> and <<worksheet,Worksheet>>.
+* It is compulsory to select the OPD ward (8) where the visit is taking place. Nevertheless, after first insertion in the session, the application will remember the latest selected to ease the daily data entry.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: I would reword: `It is compulsory to select the OPD ward (8) where the visit is taking place. Nevertheless, after first insertion in the session, the application will remember the latest selected to ease the daily data entry.` as `The selection of the the OPD ward (8)  is required so as to specify where the visit is taking place. After the first selection in a session, the application remembers the selection to ease the data entry.`

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -259,15 +261,17 @@ The OPD Extended module affords the following additional features:
...
+* if a new attendance for this patient is required click the "New Admittance" (6) check box and proceed normally.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: `if` -> `If` and add a comma after `required`

File: doc_user/UserManual.adoc [hunk 3]
```diff
@@ -259,15 +261,17 @@ The OPD Extended module affords the following additional features:
...
 * In the case the selected patient personal data need to be updated, it is possible to click on the Edit icon image:image15.png[Edit icon,width=26,height=26] near his/her name; a new window shows the patient personal information ready for modification.
 * In the case the patient is not yet registered, the *_<new patient>_* entry can be selected to register a new patient; after registration, the program returns to the OPD window and the new patient is selected.
 * If the user is enabled by the Administrator (see <<users-groups,Users & Groups>>) the following is shown:
 ** the *Examination* button function (see <<patient-examination,Patient examination>>)
-** the *Operation* tab after the patient's tab (5) to record small surgeries and other interventions (see
-<<operations,Operations>>) as well as in Admission (see <<insert-or-modify-surgery-data,Insert or Modify Surgery Data>>).
+** the *Operation* tab after the patient's tab (5) to record small surgeries and other interventions (see <<operations,Operations>>) as well as in Admission (see <<insert-or-modify-surgery-data,Insert or Modify Surgery Data>>).
+
+NOTE: for OPD and IPD wards definition, see <<ward,Ward>> settings
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: This should be a sentence: `NOTE: For OPD and IPD wards see <<ward,Ward>> settings.`

File: doc_user/UserManual.adoc [hunk 4]
```diff
@@ -2625,14 +2630,14 @@ To insert a new ward, provide the following required inputs:
...
 * *Visit duration*: length of a visit (in minutes and must be greater than zero)
-* *Ward with pharmacy*: it is used in Pharmaceuticals Stock; if checked it means that ward can be a destination for discharging movements, otherwise it will be not shown in the list
+* *Ward with OPD*: if checked, it means that the ward will also be available in the <<opd-extended,OPD Extended>> mode
+* *Ward with pharmacy*: it is used in Pharmaceuticals Stock; if checked, it means that the ward can be a destination for discharging movements, otherwise it will be not shown in the list
 * *Male Ward*: if it is a male Ward
 * *Female Ward*: if it is a female Ward
 
-NOTE: Using this function the definition of wards with 0 (zero) beds but with a pharmacy is possible.
-This allows for "logical" departments (rather than physical) to manage pharmacy movements.
+NOTE: Maternity (M) and OPD (OPD) wards are _built-in_ wards and cannot be deleted, although can be modified as preferred.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: `although can be modified as preferred.` -> `although they can be modified as preferred wards.`

**Review by dbmalkovsky — CHANGES_REQUESTED** [review]
Made some suggested changes

File: doc_user/UserManual.adoc (hunk 4 above)
Comment (mwithi) [reply doc_user/UserManual.adoc]: "as preferred" I mean "as one prefer"

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: OK; then  `although they can be modified as required.` or `although they can be modified as needed.`
